### ISSUE TITLE
Cannot be installed : No module named 'py2exe_distutils'

### ISSUE THREAD

**Issue by giuliodz** [py2exe/py2exe#84]
## Environment info
Ubuntu 20.04
Python 3.8.5

## Description

When I try to install py2exe with `pip install py2exe` I get the following error:
```
Collecting py2exe
  Downloading py2exe-0.10.3.0.tar.gz (107 kB)
     |████████████████████████████████| 107 kB 2.9 MB/s 
    ERROR: Command errored out with exit status 1:
     command: /home/giulio/anaconda3/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-993o3aq_/py2exe/setup.py'"'"'; __file__='"'"'/tmp/pip-install-993o3aq_/py2exe/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-pip-egg-info-qhsq743e
         cwd: /tmp/pip-install-993o3aq_/py2exe/
    Complete output (5 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-993o3aq_/py2exe/setup.py", line 16, in <module>
        from py2exe_distutils import Dist, Interpreter, BuildInterpreters
    ModuleNotFoundError: No module named 'py2exe_distutils'
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.

```

**Comment by albertosottile** [comment]
Hello,

`py2exe` requires Windows to work, hence it is not compatible with your environment.

However. this issue reminds me that I should adapt `setup.py` to output a more meaningful message when someone on other operative systems tries to install `py2exe` from the sdist archive. I will leave this issue open as a reminder for myself until I fix this.

**Comment by albertosottile** [comment]
Fixed in c6bbada011200d7cd1058241734be17d64a655a2
